### DISCUSSION TITLE
Two finger right click

### DISCUSSION THREAD

**Discussion by mfdorst** [hyprwm/Hyprland#921] [Q&A] [answered]
Under hyprland, my touchpad's behavior is that a click in the bottom-right-hand corner of the touchpad is a right click, and all other clicks are left clicks. I am used to a two-finger click being a right click, and I cannot find a setting to change this behavior. Maybe this can be set elsewhere? But since I found that I needed to set my trackpad's scrolling direction in my wayland config I'm assuming that this will need to be done through wayland too.

**Comment by fufexan** [comment, marked as answer]
`input:touchpad:clickfinger_behavior = 1`

It's not immediately apparent from the wiki, I'll try to change that.

**Reply by mfdorst** [reply]
Oh thanks, that worked. Yeah I saw that there were a few settings that were listed as "self explanatory" but they were not self explanatory to me lol.

**Reply by Paulie420** [reply]
OK, I'm dumb - where does the below go? hyprland.conf, or....???
input:touchpad:clickfinger_behavior = 1

**Reply by fufexan** [reply]
@Paulie420 yes, in hyprland.conf you add something like this
```ini
input {
  touchpad {
    clickfinger_behavior = 1
  }
}
```

**Reply by Paulie420** [reply]
I just figured it out. It's actually "= yes", but thanks for the reply. Maybe this will help some other newbie too. Ty.

**Reply by fufexan** [reply]
Right. In Hyprland, booleans can be `yes/no`, `0/1` or `true/false`. So either works.

**Reply by Paulie420** [reply]
Thanks for that tidbit - I didn't think 'true' worked, why I 'corrected' fufexan - sorry i was mistaken - I'll actually switch all my stuff to true/false, as that's how I best read things. Appreciate the info. (DERP.)
